schemas/workflow_runs/definitions.yaml

```diff
@@ -11,7 +11,6 @@ workflow_run_shared:
       type: array
       maxItems: 30
       description: Tags or labels assigned to the workflow run.
-      uniqueItems: true
```

Comment:
Removing `uniqueItems` since it causes generator to use `Set()` instead of an array for this property.
Side note, in JSON there is [no easy way](https://softwareengineering.stackexchange.com/questions/355176/how-to-represent-a-set-in-json) to store sets.